root/static/scripts/guess-case/MB/Control/GuessCase.js

```diff
@@ -34,9 +34,14 @@ MB.Control.initialize_guess_case = function (type, formPrefix) {
     }
 
     $name.parent()
-        .find("button.guesscase-title").on("click", function () { setVal($name, guess.guess($name.val())) })
+        .find("button.guesscase-title").on("click", function () { 
+            setVal($name, guess.guess($name.val())) 
```

Comment:
This should end on a semicolon, shouldn't it? You can check with the "semi" eslint rule (it has an auto-fix option if you run it with --fix at the end of the line).